src/test/java/org/gridsuite/actions/test/FiltersUtilsTest.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.actions.test;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Network;
+import org.gridsuite.actions.server.utils.FiltersUtils;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public class FiltersUtilsTest {
+    @Test
+    public void test() {
+        Identifiable<Generator> ident = new Identifiable<Generator>() {
```

Comment:
You could use mockito to avoid this verbose implementation

Reply:
Done